{{cookiecutter.app_name}}/docker-compose.dev.yml

```
@@ -0,0 +1,11 @@
+version: "3.5"
+
+services:
+  db:
+    image: postgres:12.3
+    container_name: {{cookiecutter.app_name}}_db
+    environment:
+      - POSTGRES_DB={{cookiecutter.app_name}}_dev
```

Prefer spelling out the complete environment name (just like for test):

```suggestion
      - POSTGRES_DB={{cookiecutter.app_name}}_development
```

---

Fixed in 337b9d9907a5dae3922b259962356bacf113f6c5